static/css/zamboni/reviewers.less

```diff
@@ -1,3 +1,6 @@
+@flagged-by-scanners-color: orange;
+@flagged-for-human-review-color: green;
+
```

Comment:
a lot of (seemingly) unnecessary reformatting in this file - these two lines at the top are the only actual changes, right?

Reply:
Indentation was inconsistent and my editor told me that it was mainly 4 spaces so I reformatted the file.. Those are the main changes indeed, along with 3 css rules similar to 3 existing ones (only the colors are different, and both colors are defined as variables)